### PR TITLE
implement php service template

### PR DESCRIPTION
**Pull request by Kravalg**

### CODE REVIEW

File: tests/Behat/DemoContext.php [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+final class DemoContext implements Context
+{
+    /** @var KernelInterface */
+    private $kernel;
+
+    /** @var Response|null */
+    private $response;
+
+    public function __construct(KernelInterface $kernel)
```
Comment (Kravalg) [inline tests/Behat/DemoContext.php:20]: I think that we need to use private read-only arguments for the constructor

**Review by Kravalg — COMMENTED** [review]
1. @Mihailmihail111 we need to add for artillery dedicated GitHub CI action.
2. @Mihailmihail111 I think that we need to add more directories inside the `src` catalog to ensure that developers will correctly understand what architecture they need to use there (according to the [PHP DDD template](https://github.com/CodelyTV/php-ddd-example#-hexagonal-architecture))

**Review by Mihailmihail111 — APPROVED** [review]

File: migrations/,gitignore
[the remark this answers is not in the document]
Comment (GritsayAndriy) [reply migrations/,gitignore:1]: You're right! And that file name is not correct.

File: tests/Behat/DemoContext.php (hunk 1 above)
Comment (GritsayAndriy) [reply tests/Behat/DemoContext.php:20]: I think it should be done.

**Review by GritsayAndriy — COMMENTED** [review]
I left a comment.

File: tests/Behat/DemoContext.php (hunk 1 above)
Comment (GritsayAndriy) [reply tests/Behat/DemoContext.php:20]: > I think that we need to use private read-only arguments for the constructor

I think it should be done.

**Comment by Kravalg** [comment]
> I left a comment.

I fixed

**Pull request merged**